docs/security/lcmaps-voms-authentication.md

```diff
@@ -208,6 +209,21 @@ Each non-commented line is a shell-style pattern which is compared against a use
 !!!warning
     `/etc/grid-security/ban-mapfile` *must* exist, even if you are not banning any users. In that case, the file should be blank. If the file does not exist, LCMAPS will ban every user.
 
+
+
+### Mapping using all FQANs
+
+By default, the LCMAPS VOMS plugin only considers for mapping the first FQAN of a VOMS proxy. This behavior matches what GUMS does. If you want to consider all FQANs, you must set the appropriate option.
+
+-   If you are using osg-configure, set `all_fqans = True` in `10-misc.ini`, then run `osg-configure -c`
+
+-   If you are configuring `lcmaps.db` manually (see [manual configuration](#manual-configuration) below), add `"-all-fqans"` to the module definitions for `vomsmapfile` and `defaultmapfile`
+
+!!! note
+    If you are using OSG 3.3, osg-configure should be at least version 1.10.2.  If you are using OSG 3.4, osg-configure should be at least version 2.2.2.
```

Comment:
This note should go at the top of the section

Reply:
Seems like a non sequitur if I put it there. What if I put it right after the bullet about osg-configure?

Reply:
Works for me